packages/components/src/public/QueryModel/ChartPanel.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FC, memo, useCallback, useMemo } from 'react';
```

Comment:
Bug. Multiple requests for the chart can result in multiple renderings of the same charts on the page. Repro:

1. Open up a JavaScript chart.
2. Throttle the network to "Slow 3G".
3. Resize the page to something smaller.
4. Notice multiple requests for the chart (Side point: It's a bit odd to be re-requesting the chart when the page resizes as the screen size is not an input to the server).

Expected: The same chart re-renders and displays appropriately for the new screen size.
Actual: Multiple charts appear.

![Screen Recording 2023-08-25 at 9 12 10 AM](https://github.com/LabKey/labkey-ui-components/assets/3926239/b45e28b9-7660-4699-95ed-9bc86b50197c)

Reply:
https://www.labkey.org/home/Developer/issues/issues-details.view?issueId=48575